support/Dockerfile

```diff
@@ -0,0 +1,51 @@
+FROM node:18-alpine AS base
+WORKDIR /opt/isomercms-backend
+
+RUN apk update && \
+    apk add --no-cache bash && \
+    apk add git && \
+    apk add openssh-client
+
+RUN adduser -u 900 webapp -D -h /home/webapp -s /bin/sh
+USER webapp
+
+COPY ../src ./src
```

Comment:
if we are spiltting this up, should we split this up for the main docker file as well?

Reply:
to clarify, we want to omit copying of `support` into our main `Dockerfile`? 

we totally can honestly, will make the change. best is if we can omit copying `src` to the support dockerfile but it doesn't impact build times much because the `src` folder is small.